File: Week-2/20-08-25 string/palindrome.c
// 1 Write a program to check whether a given string is a palindrome.
//    eg. madam
#include <stdio.h>
#include <string.h>

int main()
{
    char originalString[10] = "madam";
    char reveseString[10];

    strcpy(reveseString, originalString);
    strrev(reveseString);

    if (strcmp(originalString, reveseString) == 0)
    {
        printf("palindrome");
    }
    else
    {
        printf("Not Palindrome");
    }
}

File: Palindrome.cs
using System;
class Program
{
    static void Main()
    {
        int n = 121, r, sum = 0, temp = n;
        while (n > 0)
        {
            r = n % 10;
            sum = (sum * 10) + r;
            n /= 10;
        }
        if (temp == sum) Console.WriteLine("Palindrome");
        else Console.WriteLine("Not Palindrome");
    }
}
